main.rb

```diff
@@ -0,0 +1,158 @@
+# rubocop:disable Metrics
+require './book'
+require './classroom'
+require './rental'
+require './person'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @class = Classroom.new('class7')
+  end
+
+  def start
+    puts 'Welcome tp School Library App'
+    menu
+  end
+
+  def menu
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    user_option = gets.chomp
+    get_user_option(user_option)
+  end
+
+  def get_user_option(option)
+    case option
+    when '1'
+      all_books
+    when '2'
+      all_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      rentals_by_id
+    when '7'
+      abort('Thank you for using our Library!')
+    else
+      puts 'Please enter a number between 1 and 7'
+      puts
+      menu
+    end
+  end
+
+  def all_books
+    puts 'There are no books! Add a new book.' if @books.empty?
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    sleep 1
+    puts
+    menu
+  end
+
+  def all_people
+    puts 'The are no people! Please add a student or teacher.' if @people.empty?
+    @people.map { |person| puts "[#{person.class}] Name: #{person.name}, Id: #{person.id}, Age: #{person.age}, " }
+    sleep 1
+    puts
+    menu
+  end
+
+  def create_person
+    print 'Do you want to create a student(1) or a teacher(2)? [Input the number]:'
+    input = gets.chomp.to_i
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    case input
+    when 1
+      create_student(age, name)
+    when 2
+      create_teacher(age, name)
+    end
+    puts 'Person created successfully!'
+    menu
+  end
+
+  def create_book
+    print 'Title '
+    title = gets.chomp
+
+    print 'Author '
+    author = gets.chomp
```

Comment:
- [ ] Kindly add colons(:) to your `Title` and `Author` prints here to make your app look clean.